.luacheckrc

```diff
@@ -40,3 +40,9 @@ local test_rules = {
 
 files['test/**/*_test.lua'] = test_rules
 files['test/luatest_helpers/vtest.lua'] = test_rules
+files['test/unit-luatest/version_test.lua'] = {
+    ignore = {
+        -- Replace comparison sign
+        '581'
+    }
+}
```

Comment:
Why is it needed? Currently luacheck passes on this file even without this exception. At least when I run it locally. My version is `Luacheck: 0.23.0`. What is yours?

Reply:
Just the same as used in current patch: 0.26.1. AFAICS, it's the last one, which is available on luarocks. Tarantool's CI use the same version.

This is the output I get:
<details><summary>Luacheck 0.26.1</summary>

```
Checking test/instances/default.lua               OK
Checking test/instances/router.lua                OK
Checking test/instances/storage.lua               OK
Checking test/luatest_helpers/vtest.lua           OK
Checking test/replicaset-luatest/replicaset_3_test.lua OK
Checking test/router-luatest/router_test.lua      OK
Checking test/storage-luatest/garbage_collector_test.lua OK
Checking test/storage-luatest/storage_1_1_test.lua OK
Checking test/storage-luatest/storage_1_test.lua  OK
Checking test/storage-luatest/storage_2_test.lua  OK
Checking test/unit-luatest/config_test.lua        OK
Checking test/unit-luatest/error_test.lua         OK
Checking test/unit-luatest/util_test.lua          OK
Checking test/unit-luatest/version_test.lua       8 warnings

    test/unit-luatest/version_test.lua:213:18: 'not (x ~= y)' can be replaced by 'x == y' (if neither side is a table or NaN)
    test/unit-luatest/version_test.lua:214:18: 'not (x < y)' can be replaced by 'x >= y' (if neither side is a table or NaN)
    test/unit-luatest/version_test.lua:215:18: 'not (x > y)' can be replaced by 'x <= y' (if neither side is a table or NaN)
    test/unit-luatest/version_test.lua:222:22: 'not (x > y)' can be replaced by 'x <= y' (if neither side is a table or NaN)
    test/unit-luatest/version_test.lua:223:22: 'not (x >= y)' can be replaced by 'x < y' (if neither side is a table or NaN)
    test/unit-luatest/version_test.lua:225:22: 'not (x < y)' can be replaced by 'x >= y' (if neither side is a table or NaN)
    test/unit-luatest/version_test.lua:226:22: 'not (x <= y)' can be replaced by 'x > y' (if neither side is a table or NaN)
    test/unit-luatest/version_test.lua:231:22: 'not (x == y)' can be replaced by 'x ~= y' (if neither side is a table or NaN)

Checking vshard/cfg.lua                           OK
Checking vshard/consts.lua                        OK
Checking vshard/error.lua                         OK
Checking vshard/hash.lua                          OK
Checking vshard/heap.lua                          OK
Checking vshard/init.lua                          OK
Checking vshard/registry.lua                      OK
Checking vshard/replicaset.lua                    OK
Checking vshard/rlist.lua                         OK
Checking vshard/router/init.lua                   OK
Checking vshard/storage/init.lua                  OK
Checking vshard/storage/ref.lua                   OK
Checking vshard/storage/reload_evolution.lua      OK
Checking vshard/storage/sched.lua                 OK
Checking vshard/util.lua                          OK
Checking vshard/version.lua                       OK

Total: 8 warnings / 0 errors in 30 files
```

</details>

Reply:
This is what I get now 😎
```
gerold$ luacheck --version
Luacheck: 1.0.0
```
I suppose we can drop exact luacheck version from CI and let it install the latest one. Although it didn't find any new warnings.